dev_name: /dev/ttyTHS0
topic_name: /Quad13/Dynamics/RotorSpeedu
rsm_topic_name: /Quad13/Dynamics/RotorSpeed
imu_topic_name: /Quad13/Dynamics/IMU
camera_counter_topic_name: /Quad13/Dynamics/CAM_COUNTER
camera_data_path: /home/amov/CAMERA
exposure_time: 100
